### PR TITLE
cause tirefires

### LINKED ISSUE #3601: slack: document requirement of metabase_files channel during setup
This issue was raised on [discourse](http://discourse.metabase.com/t/slack-integration-problem/725/4). The resolution was noted in [this reply](http://discourse.metabase.com/t/slack-integration-problem/725/6).

The requirement of having a channel _metabase_files_ previously existing could be documented in both the admin guide as well as the _/admin/settings/slack_ endpoint in order to avoid any issues. Note that a bot user is unable to create a channel.

References:
https://github.com/metabase/metabase/blob/master/src/metabase/integrations/slack.clj#L75
https://api.slack.com/methods/channels.create

### PR DESCRIPTION
**Pull request by salsakran**
fixes #3601
http://tirefi.re/tirefirebillow.gif

### CODE REVIEW

File: docs/administration-guide/08-setting-up-slack.md [hunk 1]
```diff
@@ -21,7 +21,11 @@ Click over to the tab that was opened and you'll now be on the Slack Bot creatio
 
 Now give the Bot user a helpful name (we suggest `Metabot`) and click the `Add bot integration` button and a bot user will be generated for you.  Look for the Bot's API token in the next page. It will look like `xoxp-etc-etc-etc` and all you need to do is copy that value and head back to Metabase.
 
-Paste the value into the text box for `Slack API Token` and click the button to save your changes.  That's it!  Metabase will automatically run a quick test to check that the API token is working properly and if not you'll get an error message.
+Paste the value into the text box for `Slack API Token` and click the button to save your changes.  
+
+Now go to slack and create a new channel named `metabase_files`. Due to the slack api, we need this to attach graphs to pulses and Metabot answers.
+
+That's it!  Metabase will automatically run a quick test to check that the API token is working properly and if not you'll get an error message.
```
Comment (camsaul) [inline docs/administration-guide/08-setting-up-slack.md]: This is nitpicky but I would tweak this a bit for consistent capitalization, tense and punctuation. Maybe something like this?

> Now go to Slack and create a new channel named `metabase_files`. Due to limitations of the Slack API, we'll need this to attach graphs to pulses and MetaBot answers.
>
>That's it! Metabase will automatically run a quick test to check that the API token is working properly; if it isn't, you'll get an error message.

**Comment by camsaul** [comment]
This is nitpicky but I would tweak this a bit for consistent capitalization, tense and punctuation. Maybe something like this?

> Now go to Slack and create a new channel named `metabase_files`. Due to limitations of the Slack API, we'll need this to attach graphs to pulses and MetaBot answers.
>
>That's it! Metabase will automatically run a quick test to check that the API token is working properly; if it isn't, you'll get an error message.

**Comment by camsaul** [comment]
Also, don't forget to `[ci skip]` :wink:

**Comment by salsakran** [comment]
thanks for picking nits. changes made.

**Pull request merged**